Fail to build bindings: trait `From<f32>` is not implemented for `u16`

```
   Compiling magick_rust v1.0.0 (https://github.com/nlfiedler/magick-rust#f796041b)
error[E0277]: the trait bound `u16: From<f32>` is not satisfied
   --> /home/ixuta/.cargo/git/checkouts/magick-rust-325de4594d577be6/f796041/src/wand/macros.rs:144:58
    |
137 | / macro_rules! set_get_unchecked {
138 | |     ($($get:ident, $set:ident, $c_get:ident, $c_set:ident, $typ:ty )*) => {
139 | |         $(
140 | |             pub fn $get(&self) -> $typ {
...   |
144 | |                 unsafe { ::bindings::$c_set(self.wand, v.into()) }
    | |                                                          ^^^^ the trait `From<f32>` is not implemented for `u16`, which is required by `f32: Into<_>`
...   |
151 | |     }
152 | | }
    | |_- in this expansion of `set_get_unchecked!`
    |
   ::: /home/ixuta/.cargo/git/checkouts/magick-rust-325de4594d577be6/f796041/src/wand/pixel.rs:94:5
    |
94  | /     set_get_unchecked!(
95  | |         get_color_count, set_color_count, PixelGetColorCount, PixelSetColorCount,   usize
96  | |         get_index,       set_index,       PixelGetIndex,      PixelSetIndex,        f32
97  | |         get_fuzz,        set_fuzz,        PixelGetFuzz,       PixelSetFuzz,         f64
98  | |     );
    | |_____- in this macro invocation
    |
    = help: the following other types implement trait `From<T>`:
              `u16` implements `From<Char>`
              `u16` implements `From<bool>`
              `u16` implements `From<u8>`
    = note: required for `f32` to implement `Into<u16>`

For more information about this error, try `rustc --explain E0277`.
error: could not compile `magick_rust` (lib) due to 1 previous error
```

--- a/.github/workflows/test-msys2.yaml
+++ b/.github/workflows/test-msys2.yaml
@@ -1,4 +1,4 @@
-name: Run tests on Windows
+name: Run tests on Windows (MSYS2)
 
 on:
   workflow_dispatch:
@@ -14,12 +14,12 @@ jobs:
       - name: Install dependencies
         shell: C:\msys64\usr\bin\bash.exe --login '{0}'
         run: |
-          export PATH="/mingw64/bin:$PATH"
-          pacman --noconfirm -S mingw-w64-x86_64-imagemagick mingw-w64-x86_64-pkg-config
+          export PATH="/ucrt64/bin:$PATH"
+          pacman --noconfirm -S mingw-w64-ucrt-x86_64-imagemagick mingw-w64-ucrt-x86_64-pkg-config
       - uses: Swatinem/rust-cache@v2
         with:
           cache-on-failure: true
       - name: Test
         run: |
-          $env:PATH = "C:\msys64\usr\bin;C:\msys64\mingw64\bin;$env:PATH"
-          cargo test -- --skip background --skip negate_image
+          $env:PATH = "C:\msys64\usr\bin;C:\msys64\ucrt64\bin;$env:PATH"
+          cargo test -- --skip test_set_background_color

--- a/build.rs
+++ b/build.rs
@@ -113,25 +113,14 @@ impl bindgen::callbacks::ParseCallbacks for RemoveEnumVariantSuffixes {
 }
 
 fn main() {
-    let check_cppflags = if cfg!(all(target_os = "windows", not(target_env = "msvc"))) {
-        // Resolve bash from directories listed in the PATH environment variable in the
-        // order they appear.
-        Command::new("cmd")
-            .arg("/C")
-            .arg("bash")
-            .arg("MagickCore-config")
-            .arg("--cppflags")
-            .output()
-    } else {
-        Command::new("MagickCore-config")
-            .arg("--cppflags")
-            .output()
-            .or_else(|_| {
-                Command::new("pkg-config")
-                    .args(["--cflags", "MagickCore"])
-                    .output()
-            })
-    };
+    let check_cppflags = Command::new("MagickCore-config")
+        .arg("--cppflags")
+        .output()
+        .or_else(|_| {
+            Command::new("pkg-config")
+                .args(["--cflags", "MagickCore"])
+                .output()
+        });
     if let Ok(ok_cppflags) = check_cppflags {
         let cppflags = ok_cppflags.stdout;
         let cppflags = String::from_utf8(cppflags).unwrap();

--- a/src/wand/pixel.rs
+++ b/src/wand/pixel.rs
@@ -93,7 +93,7 @@ impl PixelWand {
 
     set_get_unchecked!(
         get_color_count, set_color_count, PixelGetColorCount, PixelSetColorCount,   usize
-        get_index,       set_index,       PixelGetIndex,      PixelSetIndex,        f32
+        get_index,       set_index,       PixelGetIndex,      PixelSetIndex,        bindings::Quantum
         get_fuzz,        set_fuzz,        PixelGetFuzz,       PixelSetFuzz,         f64
     );
 

